### PR TITLE
batch player summary calls when fetching user friends

### PR DESCRIPTION
**Pull request by aheadley**
the api seems to limit this to around 400 friend IDs per call, over the limit generates a BadApiCall exception

### CODE REVIEW

File: steamapi/core.py [hunk 1]
```diff
@@ -579,7 +579,18 @@ def expire(obj, property_name):
...
-        
+
+def chunker(seq, size):
+    """
+    Turn an iteratable into a iterable of iterables of size
+
+    :param seq: The target iterable
+    :type seq: iterable
+    :param size: The max size of the resulting batches
+    :type size: int
```
Comment (smiley) [inline steamapi/core.py]: Add a return-type hint as well:

```python
"""
:rtype: iterable
"""
```

**Comment by smiley** [comment]
Just a small fix needed for the docstring in `core.chunker`, but other than that, looks great! :+1:

**Comment by aheadley** [comment]
Added the return type hint.

**Comment by smiley** [comment]
Merged it in. Sorry about the wait, and thanks again!

**Pull request merged**